### DISCUSSION TITLE
more NodeGraph functionality

### DISCUSSION THREAD

**Discussion by nickper** [grafana/grafana#48957] [Dashboards, Panels & Visualizations]
I realy like the NodeGraph panel, but I want to have more control in its behavior and usability.
I was thinking about the following:
- control over edge length
- control over edge info visibility
- control over the font size of edge and node information
- partial control over the node alignment, so it doesn't turn into a jumbled mess.

And lastly it would be nice to have support from more database types out of the box.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
